code/modules/mob/living/simple_animal/bot/bot.dm

```diff
@@ -526,6 +526,8 @@ Pass a positive integer as an argument to override a bot's default speed.
 		return FALSE
 	var/step_count = move_speed ? move_speed : base_speed //If a value is passed into move_speed, use that instead of the default speed var.
 
+	path.Cut(1,2)
```

Comment:
Add an argument to get_path that runs this before returning the path, this code is currently being copy-pasted everywhere where paths are used.